src/js/lib/admin-page/index.js

```diff
@@ -1,7 +1,13 @@
 import domReady from '@wordpress/dom-ready';
 import { __, sprintf } from '@wordpress/i18n';
 
+import App from './components/App';
```

Comment:
Let's make the entry component name and path more descriptive than `App` so it's easier to grok in the editor. Maybe something like:

```suggestion
import ParselySettings from './components/ParselySettings';
```

(we'll need to `git mv ./components/App.js ./components/ParselySettings.js`, of course)

Reply:
This is done in the latest couple of commits